plugin/pkg/distributedlock/cassandra/lock.go

```diff
@@ -78,6 +79,20 @@ func (l *Lock) Acquire(resource string, ttl time.Duration) (bool, error) {
 	return false, nil
 }
 
+// Forfeit forfeits an existing lease around a given resource.
+func (l *Lock) Forfeit(resource string) (bool, error) {
+	var name, owner string
+	applied, err := l.session.Query(cqlDeleteLock, resource, l.tenantID).ScanCAS(&name, &owner)
+	if err != nil {
+		return false, errors.Wrap(err, "Failed to forfeit resource lock due to cassandra error")
+	}
+	if applied {
+		// The lock was successfully deleted
+		return true, nil
+	}
+	return false, errors.Wrap(errLockOwnership, "Failed to forfeit resource lock")
```

Comment:
Actually, I think the format is the reverse: `"Failed to forfeit resource lock: This host does not own the resource lock"`. I followed the pattern I saw for acquire/extending the lock ownership, where a similar error is returned (`"Failed to extend lease on resource lock: This host does not own the resource lock"`). See `expectedErrMsg` in `lock_test.go` for details. If this is wrong, please let me know what would be a better message.